### ISSUE TITLE
Spinning Icons

### ISSUE THREAD

**Issue by harrymcc** [petereit/CheckinAir#5]
When checking in attendees the label does print.  However, the checkbox does not show the check mark, it keeps spinning.  Only after exiting and then relaunching the application does it show the check mark.  The application does continue to run, though I have to restart occasionally due to it locking.

**Comment by petereit** [comment]
Thank you for your report. Breeze recently added the spinning icons. I'm still analyzing their changes and trying to determine the best approach to solving this new issue. Look for an app update soon.

**Comment by petereit** [comment]
I have submitted version 1.31 to the Apple App Store that fixes the spinning icon issue. It will be available for download as soon as Apple approves the update.

**Comment by harrymcc** [comment]
Thank you so much.  Have a blessed weekend.




Get Outlook for Android









On Fri, Apr 28, 2017 at 4:31 PM -0400, "Mark Petereit" <notifications@github.com> wrote:












I have submitted version 1.31 to the Apple App Store that fixes the spinning icon issue. It will be available for download as soon as Apple approves the update.
